utils/format.go

```diff
@@ -361,16 +361,7 @@ func FormatTransactionType(txnType uint8) string {
 
 // FormatCurrentBalance will return the current balance formated as string with 9 digits after the comma (1 gwei = 1e9 eth)
 func FormatCurrentBalance(balanceInt uint64, currency string) template.HTML {
-	if currency == "ETH" {
-		exchangeRate := ExchangeRateForCurrency(currency)
-		balance := float64(balanceInt) / float64(1e9)
-		return template.HTML(fmt.Sprintf("%.5f %v", balance*exchangeRate, currency))
-	} else {
-		exchangeRate := ExchangeRateForCurrency(currency)
-		balance := FormatFloat((float64(balanceInt)/float64(1e9))*float64(exchangeRate), 2)
-
-		return template.HTML(fmt.Sprintf(`%s %v`, balance, currency))
```

Comment:
The currency is now missing for the balance, you can compare your implementation with **FormatExchangedAmount**.

Reply:
added it back